db/reporting/migrate/20200306172853_create_hud_report_framework.rb

```diff
@@ -0,0 +1,25 @@
+class CreateHudReportFramework < ActiveRecord::Migration[5.2]
+  def change
+    create_table :hud_report_instances do |t|
+      t.references :user
+      t.string :name
```

Comment:
I think this needs some parameters/options as well.  They could be a somewhat generic `options` json blob, or more specific columns, or a combination.  I'm thinking a combination of `options` and `coc` since that's required and singular.  Additionally it would need some state information (started_at, completed_at, ...)